ibis/client.py

```diff
@@ -281,12 +283,12 @@ def validate_backends(backends):
 
 
 def execute(expr, limit='default', params=None, **kwargs):
-    backend, = validate_backends(list(find_backends(expr)))
+    (backend,) = validate_backends(list(find_backends(expr)))
```

Comment:
Odd that these didn't get picked up with the latest round of black formatting. have you rebased?

Reply:
not yet ... I needed to stop a little bit this task .. but I will be back on this task today or tomorrow